### PR TITLE
Update avago 1.11.10 rc0

### PR DESCRIPTION
**Pull request by ceyonur**
Fetches changes from latest Coreth release

### CODE REVIEW

File: params/config_test.go [hunk 1]
```diff
@@ -117,25 +117,25 @@ func TestCheckCompatible(t *testing.T) {
...
 			stored:        TestChainConfig,
-			new:           TestPreSubnetEVMConfig,
+			new:           TestPreSubnetEVMChainConfig,
 			headBlock:     0,
 			headTimestamp: 0,
 			wantErr: &ConfigCompatError{
 				What:         "SubnetEVM fork block timestamp",
 				StoredTime:   utils.NewUint64(0),
-				NewTime:      nil,
+				NewTime:      TestPreSubnetEVMChainConfig.NetworkUpgrades.SubnetEVMTimestamp,
```
Comment (darioush) [inline params/config_test.go:126]: why was this changed?

**Review by darioush — APPROVED** [review]

File: params/config_test.go (hunk 1 above)
Comment (ceyonur) [reply params/config_test.go:126]: `TestPreSubnetEVMChainConfig` now has a future time rather than having nils (to reflect VM behaviour)
